snapcraft/internal/deprecations.py

```diff
@@ -29,6 +29,8 @@
     'dn2': "Custom plugins should now be placed in 'snap/plugins'.",
     'dn3': "Assets in 'setup/gui' should now be placed in 'snap/gui'.",
     'dn4': "The 'history' command has been replaced by 'list-revisions'.",
+    'dn5': "Aliases are now handled by the store, and shouldn't be placed "
```

Comment:
"by the store or manually by the user" ?  not sure how important it is to convey here:
* store controls automatic aliases
* user can create manual ones but snap has no say

the deprecation note itself seems very clear, thank you

Reply:
Yeah, this is really a one-liner saying "stop doing that and do this instead" in as little space as possible. The note is where people go to understand the details.

Reply:
I think "declared" would make more sense here since they're (were) declared in the YAML and not files contained in the snap.

Reply:
Sure, declared is fine. Done.